static/styles/tui-doc-theme.css

```diff
@@ -332,45 +222,34 @@ footer {
 .readme h6 {
     font-size: 15px;
 }
-.readme table {
-    margin: 5px 30px 20px;
-}
+
 .readme table th,
 .readme table td {
-    padding: 2px 20px 2px 5px;
+
     border-bottom: 1px solid #dedede;
 }
 .readme section header h2 {
     font-size:20px;
-    padding-left:10px;
     border-left:5px solid #fa3282;
 }
 .readme section .container-overview {
     color:#333;
     border-radius: 2px;
     border:1px solid #dedede;
-    padding:15px 15px 10px;
 }
 .readme section .container-overview .description {
     color:#666;
 }
 .readme section .container-overview dt {float:left; }
 .readme section .container-overview dd {float:left; margin-left:10px; }
```

Comment:
I think managing styles separately would be good, but I wonder how to distinguish layout or theme styles strictly.

Might the distinguishments be troublesome things to contributing or custom styling.

Could you some advices?

Reply:
good point - ill respond shortly